crates/astria-composer/src/composer.rs

```diff
@@ -15,42 +26,126 @@ use crate::{
         self,
         ApiServer,
     },
-    searcher::Searcher,
+    collectors,
+    collectors::geth::GethCollector,
```

Comment:
Just import the `collectors` module and refer to the geth collectors as `collectors::Geth`